reactive_stores_macro/src/lib.rs

```diff
@@ -537,61 +537,129 @@ fn variant_to_tokens(
 struct PatchModel {
     pub name: Ident,
     pub generics: Generics,
-    pub fields: Vec<Field>,
+    pub ty: PatchModelTy,
+}
+
```

Comment:
This just wants an `#[allow(dead_code)]` for the `Enum` branch.

Reply:
Added